com.unity.render-pipelines.universal/Runtime/ScriptableRendererData.cs

```diff
@@ -36,6 +37,12 @@ public List<ScriptableRendererFeature> rendererFeatures
             get => m_RendererFeatures;
         }
 
+        /// <summary>
+        /// Controls whether the volume framework should be updated every frame by
+        /// this renderer each frame or via scripting
```

Comment:
/// Controls whether the volume framework should be updated every frame by
        /// this renderer or via scripting

Reply:
Just looks better. Doenst need to fix in this PR